edvart/report.py

```diff
@@ -121,7 +121,7 @@ def _generate_notebook(
         imports_set = {
             "import pandas as pd",
             "import os",
-            "from typing import Any, Callable, Dict, List, Optional, Tuple, Union",
+            "from typing import Any, Callable, Dict, Iterable, List, Optional, Tuple, Union",
```

Comment:
I see that `Iterable` is only used in `discrete_colorscale` which is added to report in `ParallelCoordinates.add_cells()`. Shouldn't this import be added in `ParallelCoordinates.required_imports()`?

Reply:
Moved.

I originally put it there because now we don't "normalize" the imports, so putting it in `ParallelCoordinates.required_imports()` actually adds a new line. However, it's cleaner when it's in the correct place, and with #82 this won't matter anymore since `isort` will normalize the imports.